scripts/update-changelog.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Check if the current directory is a git repository
+if ! git rev-parse --git-dir >/dev/null 2>&1; then
+  echo "Not in a git repository."
+  exit 1
+fi
+
+# Get the current git branch
+branch=$(git rev-parse --abbrev-ref HEAD)
+
+# Check if current branch is not a release branch
+if [[ ! $branch =~ ^release/ ]]; then
+  echo "Not in a release branch."
+  exit 1
+fi
+
```

Comment:
Considering that `validate-changelog.sh` can be run on or outside of a release branch, what do you think about supporting non-release branches here as well? This would make the `changelog:validate` and `changelog:update` package scripts behave the same. Perhaps we could pass `--rc` if it's a release branch, and not otherwise, similar to what `validate-changelog.sh` does.